.eslintrc

```diff
@@ -66,8 +66,8 @@
     "amphtml-internal/no-global": 0,
     "amphtml-internal/no-spread": 2,
     "amphtml-internal/no-style-property-setting": 2,
+    "amphtml-internal/prefer-deferred-promise": 1,
```

Comment:
@jridgewell You forgot to rename this rule in `.eslintrc-strict`.

https://github.com/ampproject/amphtml/blob/de9ed3a3137df89b0d0ce3372b5f0521f2632479/.eslintrc#L70

Reply:
😬